### ISSUE TITLE
It should be on ready

### ISSUE THREAD

**Issue by codex-corp** [spchuang/videojs-markers#80]
https://github.com/spchuang/videojs-markers/blob/04dff88f70c32d763d9962aeb5bc93410100e8a4/dist/videojs-markers.js#L447

````js
    player.ready(function(){
        initialize();
    });

**Comment by spchuang** [comment]
Are there cases when the current initialization method fails for you?

**Comment by abhinavatai** [comment]
> Are there cases when the current initialization method fails for you?

Yep. I've already initialized and added markers before player event ""loadedmetadata".
When you initialize it again using this event all my previous markers are removed.
This should not happen?
